handbook/product/README.md

```diff
@@ -176,16 +175,14 @@ There are two types of tooltips. The two types of tooltips have some unique styl
 
 ## Release 
 
-This section outlines the communication between the product team, marketing team, product team,
-and customer success team prior to a release of Fleet.
+This section outlines the communication between the product, marketing, and customer success teams prior to a release of Fleet.
 
 ### Goal
 
-Keep the business up to date with improvements and changes to the Fleet product so that all stakeholders can communicate
-with customers and users.
+Keep non-engineering and product departments up to date with improvements and changes to the Fleet product so that all stakeholders can communicate with customers and users.
 
 ### Product marketing tiers 
-After the kickoff of a product sprint, the marketing and product teams should decide which features are most important to highlight. When a feature gets scheduled for release the Tier helps marketing give it the biggest impact. Once the features have been discussed the next step is to bucket them into tiers for marketing prioritization. The current tiers are 1-3. 
+After the kickoff of a product sprint, the marketing and product teams should decide which features are most important to highlight. When a feature gets scheduled for release, the Tier determines the marketing effort for the feature. Once the features have been discussed the next step is to bucket them into tiers for marketing prioritization. The current tiers are 1-3. 
 
 - Tier 1: A separate product feature announcement. The most important release types, these require a specific and custom marketing package. Usually including an individual blog post, a demo video and potentially a press release or official product marketing launch. Due to limited availability there is only room for one Tier 1 product announcement per release sprint.
```

Comment:
@zhumo We could consider a noun other than "tier" here (to avoid confusion)

Reply:
@zhumo I took a pass at this here: https://github.com/fleetdm/fleet/pull/10670/files#r1144293359

Goal: Avoid confusion with pricing tiers, and make heading shorter so it makes an attractive link for sharing with fleeties, and more 🟣 approachable to community members who might not know what "product marketing" _actually_ means (I def wouldn't have known, really, not until like 2021).

tldr:
- "Tier" » "Echelon"
- "Product marketing tiers" = "Ranking features"